.coderabbit.yaml

```
@@ -0,0 +1,4 @@
+reviews:
+  profile: assertive
+  request_changes_workflow: true
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)